metrics/stats.go

```diff
@@ -94,4 +94,38 @@ var (
 			Help:      "Bucketed histogram of some stats in fast analyze.",
 			Buckets:   prometheus.ExponentialBuckets(1, 2, 16),
 		}, []string{LblSQLType, LblType})
+
+	SyncLoadCounter = prometheus.NewCounter(
```

Comment:
We haven't updated the `tidb.json` for the grafana.
Would we update it in this pr together?

Reply:
It's in another PR, https://github.com/pingcap/tidb/pull/30812. I see other people update it in separate PR.